pkg/chartRepo/repository/ChartRepoRepository.go

```diff
@@ -303,6 +303,7 @@ type ChartRefRepository interface {
 	GetDefault() (*ChartRef, error)
 	FindById(id int) (*ChartRef, error)
 	GetAll() ([]*ChartRef, error)
+	DataExists(name string, version string) (bool, error)
```

Comment:
can we change this method name to CheckIfDataExists or something like this for clearer understanding?